README.md

```diff
@@ -31,6 +32,7 @@ A Typescript Action that performs HTTP requests within your workflow. It support
   uses: CamiloGarciaLaRotta/watermelon-http-client@v1
   with:
     url: 'https://countries.trevorblades.com/'
+    headers: '{"Content-Type": "application/json"}'
```

Comment:
🤔For this examples section I was thinking of giving the minimal copy-pasteable snippets that work out-of the box. In which case I think we can remove this line:

```suggestion
```

One thing that is nice so far is that we have "sane defaults", meaning that sending no custom headers, defaults to the appropriate GraphQL headers. You can see that it succeed in the [GraphQL workflow](https://github.com/CamiloGarciaLaRotta/watermelon-http-client/runs/562788947#step:3:21) without us needing to defining the headers:
https://github.com/CamiloGarciaLaRotta/watermelon-http-client/blob/f2870ebecac7918a018b83fa86c7e04ec3452047/.github/workflows/graphql.yml#L17-L25

Reply:
On the other hand, I see the point of giving an example of how to define custom headers. When we merge https://github.com/CamiloGarciaLaRotta/watermelon-http-client/pull/26 we can refer to that file as an example directly 👍🏽

Reply:
I agree. Removed !